pkg/operation/botanist/kubeapiserver_test.go

```diff
@@ -640,17 +639,19 @@ var _ = Describe("KubeAPIServer", func() {
 		})
 	})
 
-	Describe("#ScaleKubeAPIServerToOne", func() {
-		deployment := &appsv1.Deployment{ObjectMeta: metav1.ObjectMeta{Name: "kube-apiserver", Namespace: seedNamespace}}
+	// TODO(acumino): Enable this test again when fakeclient start supporting scale subresource.
```

Comment:
Is there an upstream issue or PR that we could link here?

Reply:
I have linked the issue in TODO. PTAL